src/lib/context/crudContext.ts

```diff
@@ -0,0 +1,69 @@
+import { Operation } from './operation';
+
+export interface BaseCrudContext<T, C> {
+    /** Http context */
```

Comment:
Is there any reason to use block comment style for oneline comments on this place, and a single line comment for oneline on l.13?

Reply:
@smoliji Aye, this is not just a block comment, but also a documentation magic :sparkles: comment.
I understand it look weird, but typedoc (or generally any documentation tool) generally skips normal comments and rightfully so.